docs/_steps/adding_editing_workflow_step.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Adding or editing workflow steps
+lang: en
+slug: adding-editing-steps
+order: 3
+beta: true
+---
+
+<div class='section-content'>
+
+When a builder adds (or later edits) your step in their workflow, your app will receive a `workflow_step_edit` action. The callback assigned to the `edit` property of the `WorkflowStep` configuration object passed in during instantiation will run when this action occurs.
+
+Whether a builder is adding or editing a step, you need to provide them with a special `workflow_step` modal — a workflow step configuration modal — where step-specific settings are chosen. Since the purpose of this modal is tied to a workflow step's configuration, it has more restrictions than typical modals—most notably, you cannot include `title​`, `submit​`, or `close`​ properties in the payload.
+
+By default, the `callback_id` for this modal will be the same as that of the workflow step. For more advanced use cases, you can override this by setting an additional property on the `WorkflowStep` configuration object, `viewWorkflowStep`, with an alternative `callback_id`.
```

Comment:
This reference to advanced use cases should be taken out. I removed the ability to provide an alternative, custom `callback_id` for the `view_submission` event because there are no plans to support multi-view/stacked modals (which was the reason it was introduced in the first place) and I cannot think of a reason that being able to do so would be necessary.